code/modules/jobs/job_types/prisoner.dm

```diff
@@ -0,0 +1,25 @@
+/datum/job/prisoner
+	title = "Prisoner"
+	flag = PRISONER
+	department_head = list("Head of security")
+	department_flag = CIVILIAN
+	faction = "Station"
+	total_positions = 3
```

Comment:
3 seems a bit much. 2 seems more reasonable. You want to have at least one free cell that security can use for their own stuff.